### DISCUSSION TITLE
getStaticProps with dynamic data

### DISCUSSION THREAD

**Discussion by guykogel** [vercel/next.js#11064] [Help]
My goal is to use getStaticProps with data that pulls from CDN (The data changes on daily basis).
The days should be pre-rendered.

Is it possible?

**Comment by heyitsarpit** [comment]
`getStaticProps` runs at build time, so if your data changes at daily basis then you have to push a new build daily.
If you don't want to do that, then just use the static data as a placeholder(the UI won't be empty when the page loads but updates quickly after first update.)
